.gitmodules

```diff
@@ -2,11 +2,14 @@
 	path = third_party/habitat-lab
 	url = git@github.com:naokiyokoyama/habitat-lab.git
 	branch = pvp
+	ignore = dirty
```

Comment:
what does `dirty` here mean?

Reply:
So when we install our third-party dependencies with `pip install -e .` it creates installation/build files and folders within the dep folder, e.g. `third-party/XYZ/XYZ.egg` etc. In the current submodule setup, these files are considered "dirty" and show up as changes or modifications when one does `git status`.

By adding `ignore = dirty` in submodules, we hide these changes from parent git tracking, i.e. `git status` will now not show third-party folders as modified. We've seen instances of people committing unwanted changed to submodule tracking (in other repos), this will reduce the chances of that happening in this repo.

Reply:
Add a note in readme so people can track their changes